src/actions/guides/database/models.cr

```diff
@@ -407,19 +424,22 @@ class Guides::Database::Models < GuideAction
       table do
         column name : String
         has_many taggings : Tagging
-        has_many posts : Post, through: :taggings
+        has_many posts : Post, through: [:taggings, :post]
       end
     end
 
     class Post < BaseModel
       table do
         column title : String
         has_many taggings : Tagging
-        has_many tags : Tag, through: :taggings
+        has_many tags : Tag, through: [:taggings, :tag]
       end
     end
     ```
 
+    The `through` is always set to an array like in the example `[:taggings, :tag]`. The first item `:taggings` is a Symbol name
```

Comment:
This was actually kind of hard to explain. I'm hoping it just "makes sense" to people, but if anyone has a better way to word this without getting "too wordy", I'm open.

Reply:
You might read through Ecto's docs on this and see if anything they have is more clear https://hexdocs.pm/ecto/Ecto.Schema.html#has_many/3-has_many-has_one-through

Reply:
Ok, I updated that. Just literally copied what they had because that was pretty clear. It's a bit more wordy than I wanted, but as long as people get it, that's all that really matters.